apps/typegpu-docs/src/content/examples/rendering/3d-fish/render.ts

```diff
@@ -132,6 +145,7 @@ export const fragmentShader = tgpu['~unstable']
       const hsv = rgbToHsv(desaturatedColor);
       hsv.y += desaturationFactor / 2;
       hsv.z += desaturationFactor;
+      hsv.x += (input.variant - 0.5) * 0.2;
```

Comment:
Tiny nit, let's add some documentation here
```suggestion
      // Hue shift
      hsv.x += (input.variant - 0.5) * 0.2;
```